src/main/scala/com/microsoft/partnercatalyst/fortis/spark/transforms/sentiment/WordListSentimentDetector.scala

```diff
@@ -0,0 +1,77 @@
+package com.microsoft.partnercatalyst.fortis.spark.transforms.sentiment
+
+import java.io.{File, IOError, IOException}
+import java.util.concurrent.ConcurrentHashMap
+
+import com.microsoft.partnercatalyst.fortis.spark.logging.Loggable
+import com.microsoft.partnercatalyst.fortis.spark.transforms.ZipModelsProvider
+import com.microsoft.partnercatalyst.fortis.spark.transforms.nlp.Tokenizer.tokenize
```

Comment:
Importing a specific function like this globally seems a little misleading. If you change `tokenize` to `apply` in `Tokenizer`, then you can tokenize in this file with `Tokenizer("input string")`